functions/src/plugins/size.ts

```diff
@@ -238,7 +238,7 @@ export class SizeTask extends Task {
     ));
 
     return sizeArtifacts.firestore.runTransaction(async transaction => {
-      const results = await transaction.getAll(...artifactDocs);
+      const results = await transaction.getAll(...artifactDocs as [any]);
```

Comment:
I had to force this type to any, because Typescript throws an error saying that it "Expected at least 1 arguments, but got 0 or more.". Apparently it doesn't handle length on spread arrays very well.